scripts/README.md

```diff
@@ -0,0 +1,11 @@
+# Helper scripts
+ 
+## Export comments (export_comments.py)
+### Description
+This is used to get all the comments (q_code 146) from a survey for a given period.  It reads the comments and generates
+an excel file with them in.
+ 
+### Usage
+ - Get the survey id and period that you wish to see the comments for
+ - Run the script with ```python3 export_comments.py <survey_id> <period>``` (assuming you're in a virtual environment that has been set up correctly)
```

Comment:
That's fair.... but most of the time you'll be running this script in preprod/prod and would already have a virtual environment set up for you.  Also, if you're running this script then there are very good odds that you know about store and how to set it up (it's described in the README).  
I'll leave it the way it is because this is mostly meant to be a hint on how to run it if you forget, rather then a guide on how to run it from nothing